plugins/qe-agent/skills/SKILL.md

```
@@ -0,0 +1,432 @@
+  "env": {
+    "MULTISTAGE_PARAM_OVERRIDE_OTEL_TESTS_BRANCH": "rhosdt-3.9"
+  }
+}
+```
+
+- `step_script_ref` — path relative to `ci-operator/step-registry/` in the openshift/release repo
+- `env` — runtime env var values that were injected at job time and are needed to reproduce setup (e.g. branch names, image refs); most steps have an empty `env`
+
+Construct the raw GitHub URL and fetch the script:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)